modules/process/manager.go

```diff
@@ -6,13 +6,12 @@ package process
 
 import (
 	"bytes"
+	"context"
```

Comment:
Next LTS is [18.04](https://www.ubuntu.com/info/release-end-of-life), which should include 1.7+. IMO we should wait for that until droping support for 1.6